src/modules/context2d.js

```diff
@@ -1277,7 +1277,12 @@
         var scaleFactorY = 1;
 
         if (typeof swidth !== 'undefined' && typeof width !== 'undefined') {
-            isClip = true;
+
+            if(swidth != imageProperties.width || sheight != imageProperties.height) {
```

Comment:
Use the clipping method to achieve it.

Reply:
I'm not sure how clipping works, but wouln't the whole image then be saved in the pdf !?